app/config.py

```diff
@@ -216,6 +217,9 @@ class Config(object):
     SQLALCHEMY_POOL_RECYCLE = 300
     SQLALCHEMY_ECHO = env.bool("SQLALCHEMY_ECHO", False)
     PAGE_SIZE = 50
+    PERSONALISATION_SIZE_LIMIT = env.int(
+        "PERSONALISATION_SIZE_LIMIT", 1024 * 120
```

Comment:
You mention 50kb in the PR description but its set to 120kb here.

Reply:
Yeah it's true! I didn't push that change yet. I want to meet with the Forms team before setting on a definitive value (meeting set on Monday), as they make use of this one-var template pattern. I could get their feedback before moving on. 

I will update the PR to 120k though in the meantime.